grails-shell/build.gradle

```diff
@@ -21,25 +21,25 @@ dependencies {
     compileOnly "org.codehaus.groovy:groovy-ant:$GroovySystem.version"
     compileOnly "org.codehaus.groovy:groovy-json:$GroovySystem.version"
 
-    runtimeOnly "org.apache.groovy:groovy-ant:$groovyVersion"
-    runtimeOnly "org.apache.groovy:groovy-json:$groovyVersion"
-    runtimeOnly "org.apache.groovy:groovy-jmx:$groovyVersion"
-    runtimeOnly "org.apache.groovy:groovy-xml:$groovyVersion"
+    runtimeOnly "org.apache.groovy:groovy-ant"
+    runtimeOnly "org.apache.groovy:groovy-json"
+    runtimeOnly "org.apache.groovy:groovy-jmx"
+    runtimeOnly "org.apache.groovy:groovy-xml"
 
     api project(":grails-bootstrap")
     api project(":grails-gradle-model")
-    api "org.apache.ant:ant:$antVersion"
-    api "org.fusesource.jansi:jansi:$jansiVersion"
-    api "jline:jline:$jlineVersion"
+    api "org.apache.ant:ant"
+    api "org.fusesource.jansi:jansi"
+    api "jline:jline"
 
     compileOnly "org.gradle:gradle-tooling-api:$gradleToolingApiVersion"
-    compileOnly "org.springframework.boot:spring-boot:$springBootVersion"
-    compileOnly "org.springframework.boot:spring-boot-loader-tools:$springBootVersion"
-    compileOnly "org.springframework:spring-web:$springVersion"
-    api "org.springframework.boot:spring-boot-cli:$springBootVersion", {
+    compileOnly "org.springframework.boot:spring-boot"
+    compileOnly "org.springframework.boot:spring-boot-loader-tools"
+    compileOnly "org.springframework:spring-web"
+    api "org.springframework.boot:spring-boot-cli", {
         exclude group: "org.codehaus.groovy", module: "groovy"
```

Comment:
I doubt `org.codehaus.groovy:groovy` is included in `org.springframework.boot:spring-boot-cli:3.3.4`

Reply:
removed